build/migration-guide.md

```diff
@@ -0,0 +1,109 @@
+# Migration to PoS user guide
+
+These instructions are exclusively for users interested in using Nimiq’s Migration Tools to transition from PoW to PoS. If you wish to become a validator post-migration, please refer to this [documentation](becoming-a-validator.md). This is the user guide for the registration tool. For a complete understanding of the migration technicalities, refer to this [document](/learn/protocol/migration).
+
+## Validator Registration Tool
+
+The Validator Registration tool facilitates the registration process for users migrating from PoW to PoS during the pre-registration phase. Users register within the PoW chain. The Validator Registration tool operates in two distinct modes:
+
+- Address and corresponding keys generation
+- Validator registration via transactions
+
+ First, you need to clone the [core-js repository](https://github.com/nimiq/core-js?tab=readme-ov-file#quickstart) and follow the instructions.
+
+### Step 1: Generate the keys
+
+Execute the script without parameters to generate the validator address, signing key, and voting key:
+
+```shell
+node validator-registration.js
+```
+
+The tool generates the keys and pastes them into the `validator-keys.json` file. The screenshot demonstrates the output of the script.
+
+<img class="object-contain max-h-[max(80vh,220px)]" src="/assets/images/migration/migration.png" alt="skip block struct" />
+
+<aside>
+💡 Save the private key securely! There is no recovery mechanism for lost private keys. Once lost, access to your validator and NIM may be permanently lost.
+
+</aside>
+
+### Step 2: Fund your address
+
+To start the validator registration process, fund the address you just generated to cover the nominal transaction fees of 1 Luna each (you need 6 Lunas) associated with the validator generation process. You can use any wallet to fund this address.
+
+### Step 3: Execute the Validator Registration Script
+
+::: details Testnet {open}
+
+For testing this in the PoW testnet, you must specify the network argument as follows:
+
+```shell
+node validator-registration.js --validator validator-keys.json --network test
+```
+
+:::
+
+::: details Mainnet
+
+Run the following script with your address and public keys as parameters:
+
+```shell
+node validator-registration.js --validator validator-keys.json
```

Comment:
```suggestion
node validator-registration.js --validator validator-keys.json --network main
```

The `test` network is the default network if none is provided.

Reply:
@Eligioo, this is not needed for the test run, right?

Reply:
Correct as it has been decided that the testnet is the default network for the script.